CMakeLists.txt

```diff
@@ -63,7 +63,7 @@ option(CMAKE_FIND_PACKAGE_PREFER_CONFIG
 set(CMAKE_FIND_FRAMEWORK NEVER)
 
 # Library requirements.
-find_package(SDL2 2.0.18 REQUIRED)
+find_package(SDL2 2.24 REQUIRED)
```

Comment:
Nope, just a check around that specific command, please. 

https://wiki.libsdl.org/SDL2/SDL_VERSION_ATLEAST